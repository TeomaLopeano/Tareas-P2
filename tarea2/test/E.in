cadenaSiguiente 0
insertarAlInicio 5 5.0
cadenaSiguiente 1
insertarAlInicio 5 5.0
cadenaSiguiente 1
insertarAlInicio 3 3.0
cadenaSiguiente 0
cadenaSiguiente 1
cadenaSiguiente 2
insertarAlInicio 13 13.0
insertarAlInicio 25 25.0
insertarAlInicio 8 8.0
insertarAlInicio 6 6.0
insertarAlInicio 14 14.0
insertarAlInicio 9 9.0
insertarAlInicio 0 0.0
insertarAlInicio 19 19.0
insertarAlInicio 33 33.0
cadenaSiguiente 0
cadenaSiguiente 1
cadenaSiguiente 11
cadenaSiguiente 12
Fin