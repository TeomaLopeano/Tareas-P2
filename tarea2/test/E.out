1>
2>Se insertó (5,5.00) al inicio de la cadena.
3>(5,5.00)
4>Se insertó (5,5.00) al inicio de la cadena.
5>(5,5.00)(5,5.00)
6>Se insertó (3,3.00) al inicio de la cadena.
7>(3,3.00)(5,5.00)(5,5.00)
8>(5,5.00)(5,5.00)(3,3.00)
9>(5,5.00)(3,3.00)(5,5.00)
10>Se insertó (13,13.00) al inicio de la cadena.
11>Se insertó (25,25.00) al inicio de la cadena.
12>Se insertó (8,8.00) al inicio de la cadena.
13>Se insertó (6,6.00) al inicio de la cadena.
14>Se insertó (14,14.00) al inicio de la cadena.
15>Se insertó (9,9.00) al inicio de la cadena.
16>Se insertó (0,0.00) al inicio de la cadena.
17>Se insertó (19,19.00) al inicio de la cadena.
18>Se insertó (33,33.00) al inicio de la cadena.
19>(33,33.00)(19,19.00)(0,0.00)(9,9.00)(14,14.00)(6,6.00)(8,8.00)(25,25.00)(13,13.00)(3,3.00)(5,5.00)(5,5.00)
20>(19,19.00)(0,0.00)(9,9.00)(14,14.00)(6,6.00)(8,8.00)(25,25.00)(13,13.00)(3,3.00)(5,5.00)(5,5.00)(33,33.00)
21>(5,5.00)(33,33.00)(19,19.00)(0,0.00)(9,9.00)(14,14.00)(6,6.00)(8,8.00)(25,25.00)(13,13.00)(3,3.00)(5,5.00)
22>(33,33.00)(19,19.00)(0,0.00)(9,9.00)(14,14.00)(6,6.00)(8,8.00)(25,25.00)(13,13.00)(3,3.00)(5,5.00)(5,5.00)
23>Fin.
